go/service/config.go

```diff
@@ -303,7 +303,7 @@ func mergeIntoPath(g *libkb.GlobalContext, p2 string) error {
 
 func (h ConfigHandler) HelloIAm(_ context.Context, arg keybase1.ClientDetails) error {
 	tmp := fmt.Sprintf("%v", arg.Argv)
-	re := regexp.MustCompile(`\b(chat|encrypt|git)\b`)
+	re := regexp.MustCompile(`\b(chat|pgp encrypt|encrypt|git|team accept-invite)\b`)
```

Comment:
Just `accept-invite` rather than `team accept-invite`, in case an option slips in there.

Reply:
ah you're right, better not to be smart with this, just catch everything